frontend/src/shared/store/actions.js

```diff
@@ -113,9 +113,19 @@ export default (moduleName, moduleService = null) => {
           );
           commit('CREATE_SUCCESS', response);
 
-          Message.success(
-            i18n(`entities.${moduleName}.create.success`),
-          );
+          if (moduleName === 'member') {
+            const contactId = response.id;
+            const successMessageAction = i18n(`entities.${moduleName}.create.message`);
+            const message = `<button class="btn btn--xs btn--bordered">
+                                <a href="/contacts/${contactId}" class="text-current">${successMessageAction}</a>
+                             </button>`;
```

Comment:
Could you update the message to be like?
```
const message = h(
              'el-button',
              {
                class: 'btn btn--xs btn--bordered !h-6',
                onClick: () => {
                  router.push({
                    name: 'memberView',
                    params: { id: contactId },
                  });
                  Message.closeAll();
                },
              },
              successMessageAction,
            );
```

It will allow to have better control on what's displayed, with the correct styling and will close the Toast message after user is redirected :smile:

Reply:
Ahh thank you for this. I knew the experience wasn't great but didn't know how to use the router in the HTML template since it treats it as a string :)